.github/workflows/nginxtest.yml

```diff
@@ -0,0 +1,31 @@
+name: Nginx tests
+
+on:
+  push:
+    branches:
+      - unstable
+      - hotfixes
+      - master
+  pull_request:
+
+jobs:
+  pre_job:
+    name: Path match check
+    runs-on: ubuntu-latest
+    # Map a step output to a job output
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
```

Comment:
This doesn't appear to be doing anything with `should_skip` and hence skipping the build if unnecessary